stages/test/org_osbuild_kickstart.py

```diff
@@ -0,0 +1 @@
+../org.osbuild.kickstart
```

Comment:
I'd prefer not to have a symlink under a different name, see later comment about `importlib`.